packages/core/components/InputOrGroup/styles.module.css

```diff
@@ -33,109 +32,143 @@
 }
 
 .Input-labelIcon {
-  color: var(--puck-color-grey-6);
+  color: var(--puck-color-grey-07);
   margin-right: 4px;
   padding-left: 4px;
 }
 
 .Input-disabledIcon {
-  color: var(--puck-color-grey-4);
+  color: var(--puck-color-grey-05);
   margin-left: auto;
 }
 
 .Input-input {
-  background: white;
+  background: var(--puck-color-white);
   border-width: 1px;
   border-style: solid;
-  border-color: var(--puck-color-grey-8);
+  border-color: var(--puck-color-grey-09);
   border-radius: 4px;
   font-family: inherit;
   font-size: 14px;
   padding: 12px 15px;
+  transition: border-color 50ms ease-in;
   width: 100%;
 }
 
 select.Input-input {
-  /* Safari */
-  appearance: none;
+  appearance: none; /* Safari */
   background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%23c3c3c3'><polygon points='0,0 100,0 50,50'/></svg>")
     no-repeat;
   background-size: 12px;
   background-position: calc(100% - 12px) calc(50% + 3px);
   background-repeat: no-repeat;
-  background-color: white;
+  background-color: var(--puck-color-white);
+  cursor: pointer;
 }
 
-.Input--readOnly > .Input-input,
-.Input--readOnly > select.Input-input {
-  background-color: var(--puck-color-grey-11);
-  border-color: var(--puck-color-grey-8);
-  color: var(--puck-color-grey-5);
-  opacity: 1;
+@media (hover: hover) and (pointer: fine) {
+  .Input:has( > input):hover .Input-input:not([readonly]),
+  .Input:has( > textarea):hover .Input-input:not([readonly]) {
+    border-color: var(--puck-color-grey-05);
+    transition: none;
+  }
+  .Input:has( > select):hover .Input-input:not([disabled]) {
+    background-color: var(--puck-color-azure-12);
+    background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%235a5a5a'><polygon points='0,0 100,0 50,50'/></svg>");
+    border-color: var(--puck-color-grey-05);
+    transition: none;
+  }
 }
 
-.Input-input:hover {
-  border-color: var(--puck-color-neutral-3);
+.Input-input:focus {
+  border-color: var(--puck-color-grey-05);
+  outline: 2px solid var(--puck-color-azure-05);
+  outline-offset: 2px;
+  transition: none;
 }
 
-.Input-input:focus {
-  border-color: var(--puck-color-azure-4);
-  outline: var(--puck-color-azure-8) 4px solid;
-  outline-offset: 0;
+.Input--readOnly > .Input-input,
+.Input--readOnly > select.Input-input {
+  background-color: var(--puck-color-grey-10);
```

Comment:
I personally find the background (`grey-10`) and color (`grey-04`) a bit visually intense for the readOnly state. Additionally, I considered that the padlock icon counted as an additional indicator to the user.

Are you trying to fix an a11y issue here or are you trying to conform to the color system, or both?

How would using these slightly softer values work? They increase the contrast from the existing values, but do so a bit more subtly.

```css
  background-color: var(--puck-color-grey-11);
  color: var(--puck-color-grey-05);
```

Reply:
a11y issue. I think the text 100% needs to be percievable, for example when external input is being used it's not really a disabled control, it's a populated-but-read-only control, so it needs to be able to be read IMO so the user can reason about it.

`grey-05` on `grey-11` does not have sufficent contrast, but `grey-04` on `grey-11` would have if that helps.

Reply:
`04` on `11` would work great!

Reply:
Updated